### PR TITLE
Imported WechatUnrecalled into DF Devspaces

### PR DESCRIPTION
**Pull request by rafaelbogaveev**

### CODE REVIEW

**Comment by dimazaur** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

**Comment by dimazaur** [comment]
This Pull Request contains more added lines than expected. Therefore, it will not be reviewed.

File: app/src/main/java/com/fkzhang/wechatunrecalled/Adapter/SupportListAdapter.java [hunk 1]
```diff
@@ -1,64 +1,64 @@
-package com.fkzhang.wechatunrecalled.Adapter;
-
```
Comment (maveco69) [inline app/src/main/java/com/fkzhang/wechatunrecalled/Adapter/SupportListAdapter.java]: hmm what is this ?

File: devspaces/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
+FROM ubuntu
+
+LABEL description="This file generates a docker image containing everything needed to build  WechatUnrecalled project."
+
+# Install JDK 8
+RUN apt-get update \
+ && apt-get install -y openjdk-8-jdk wget unzip git sudo
```
Comment (maveco69) [inline devspaces/docker/Dockerfile]: not cleaning in the end

Comment (rafaelbogaveev) [reply devspaces/docker/Dockerfile]: done

File: app/src/main/java/com/fkzhang/wechatunrecalled/Adapter/SupportListAdapter.java (hunk 1 above)
Comment (maveco69) [reply app/src/main/java/com/fkzhang/wechatunrecalled/Adapter/SupportListAdapter.java]: Fixing builds in this case (was not working, but usually as a customer we should not touch), I will approve this as it is a trial.

**Pull request merged**